iamy.go

```diff
@@ -18,6 +18,7 @@ var (
 	defaultDir      string
 	dryRun          *bool
 	versionFileName string = ".iamy-version"
+	configFileName  string = ".iamy-config"
```

Comment:
I'm wondering if this should this be called `.iamy-flags`, to better suit it's contents:

```
❯ cat .iamy-config
--skip-tagged=iamy-ignore
```

Reply:
Sure works for me.